virify.nf

```diff
@@ -549,12 +582,16 @@ def helpMSG() {
     --length            Initial length filter in kb [default: $params.length]
     --sankey            select the x taxa with highest count for sankey plot, try and error to change plot [default: $params.sankey]
     --chunk             WIP: chunk FASTA files into smaller pieces for parallel calculation [default: $params.chunk]
+    --onlyannotate      Only annotate the input FASTA (no virus prediction, only contig length filtering) [default: $params.only_annotate]
 
     ${c_yellow}Developing:${c_reset}
-    --version         define the ViPhOG db version to be used [default: $params.version]
-                      v1: no additional bit score filter (--cut_ga not applied, just e-value filtered)
-                      v2: --cut_ga, min score used as sequence-specific GA, 3 bit trimmed for domain-specific GA
-                      v3: --cut_ga, like v2 but seq-specific GA trimmed by 3 bits if second best score is 'nan'
+    --viphog_version    define the ViPhOG db version to be used [default: $params.viphog_version]
+                        v1: no additional bit score filter (--cut_ga not applied, just e-value filtered)
+                        v2: --cut_ga, min score used as sequence-specific GA, 3 bit trimmed for domain-specific GA
+                        v3: --cut_ga, like v2 but seq-specific GA trimmed by 3 bits if second best score is 'nan'
+    --meta_version      define the metadata table version to be used [default: $params.meta_version]
+                        v1: older version of the meta data table using an outdated NCBI virus taxonomy 
+                        v2: 2020 version of NCBI virus taxonomy
```

Comment:
Do we need the old version of the NCBI virus taxonomy?.

Reply:
This is exactly the pain point Guillermo and I are working on. We currently achieve more taxon assignments with the metadata table v1 based on the old NCBI virus taxonomy, although the v2 is more complete. We investigate, if some of the additional assignments with v1 are just false positives. And yes, the idea is to simply use the updated v2 of this table. 

For now, it's easier for me to test v1 vs v2 with this flag :)

Reply:
Cool :)